behaviors/simple-list.js

```diff
@@ -146,6 +146,8 @@ module.exports = function (result, args) {
       // returns false if repitition is allowed
       // returns false if repitition is disallowed and items don't repeat
       function hasRepeatedValue(value, data) {
+        //if data is undefined make it an empty array
+        var data = !data ? [] : data;
```

Comment:
this should probably be done above in the `bind` function (it's where the data initializes). Also, keep in mind our linter doesn't like multiple `var` declarations 😄